packages/fuels-signers/src/provider.rs

```diff
@@ -284,7 +289,12 @@ impl Provider {
                      owner: _,
                      amount,
                      asset_id,
-                 }| (asset_id.to_string(), amount.try_into().unwrap()),
+                 }| {
+                    (
+                        AssetId::from_str(&asset_id.to_string()).unwrap(),
```

Comment:
Is there a reason why this isn't just `asset_it`

Reply:
Yes. `AssetId` here is `fuels_types::AssetId` and `asset_id` is `fuel_gql_client::client::schema::AssetId`. There is no direct conversion.